examples/jupyter/demo010_advection.ipynb

```diff
@@ -0,0 +1,149 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Advection Example"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# dependencies\n",
```

Comment:
This portion was updated to
```
# dependencies
using Pkg
Pkg.activate(".")
Pkg.instantiate()
Pkg.develop(path="../..")
using Plots
using LFAToolkit
using LinearAlgebra
```

Reply:
I have this  
```
using Pkg
Pkg.activate("./")
Pkg.instantiate()
Pkg.develop(path="./../../../LFAToolkit.jl")
using Plots
using LFAToolkit
using LinearAlgebra 
using Polynomials
```

Why are we not using ```Polynomials``` anymore? We will need that when invoking the ```sausagetransformation```

Reply:
We're using `Polynomials` inside of the library. I'm talking about the first block in the Jupyter notebook

Reply:
Ok, I will change the path right away

Reply:
Yeah, we aren't calling `Polynomials` here, it's just used by the library.